packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -20,9 +14,13 @@ export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
   return {
     root: [
       classNames.root,
-      getFocusStyle(theme, -2),
       {
-        color: semanticColors.link
+        color: semanticColors.link,
+        selectors: {
+          '.ms-Fabric--isFocusVisible &:focus': {
```

Comment:
you're missing a default of `outline: none`. This is causing the screener test to fail.

outline should be none, until you're focused and have the right root classname defined.

Reply:
ah thank you! i've been trying to figure out the screener failure

Reply:
is there a way to locally run screener tests? or can we only trigger them by pushing? trying to see if i can re-run and confirm the fix before pushing

Reply:
> 
> 
> is there a way to locally run screener tests? or can we only trigger them by pushing? trying to see if i can re-run and confirm the fix before pushing

Yes you can, see https://github.com/OfficeDev/office-ui-fabric-react/blob/master/apps/vr-tests/package.json#L11

https://github.com/OfficeDev/office-ui-fabric-react/pull/6336#issuecomment-420795399